src/adapters/axiosAdapter.ts

```diff
@@ -12,12 +13,10 @@ export class AxiosAdapter<TResource> implements Adapter<TResource> {
                 url: options.url,
                 maxContentLength: Infinity,
                 maxBodyLength: Infinity,
+                // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
Why did these light up?

Reply:
Yeah not sure why they wouldn't have turned up before, as soon as I opened the file it lit up. We have these in a bunch of places across our various typescript codebases where we need them, so I think the disable is fine in an of itself.